igc_lib.py

```diff
@@ -58,28 +58,38 @@ def rawtime_float_to_hms(timef):
     
     return hms((time/3600), (time%3600)/60, time%60)
 
-def degrees_float_to_degrees_minutes_seconds(dd):
-    """Converts time from floating point degrees to degrees/minutes/floating point seconds.
+def degrees_float_to_degrees_minutes_seconds(dd, lon_or_lat):
```

Comment:
Could you also update the `dump_thermals_to_cup_file()` function? It calls `degrees_float_to_degrees_minutes_seconds()`.